opentech/apply/funds/views.py

```diff
@@ -153,8 +155,15 @@ def get_context_data(self, **kwargs):
         if self.object.next:
             other_submissions = other_submissions.exclude(id=self.object.next.id)
 
+        try:
+            public_page = BaseApplicationPage.objects.get(application_type_id=self.object.page.id)
```

Comment:
You should be able to use `self.object.get_from_parent('detail')()` which exists for both labs and applications

https://github.com/OpenTechFund/opentech.fund/blob/9fd79c7135d8e3fb003c0abe8418f1cac68b0438/opentech/apply/funds/models/applications.py#L60-L62

Reply:
Works perfectly, thanks.